assets/js/app.js

```diff
@@ -24,78 +24,6 @@ let header = $(`
    <li class="nav-item">
    <input type="checkbox" class="dark_toggler" aria-label="Toggle Light Mode" onclick="toggle_light_mode()">
    </li>
-   <div class="bike">
-   <svg xmlns="http://www.w3.org/2000/svg" viewBox="-80 0 650 400" preserveAspectRatio="xMinYMin meet">
-     <rect/>
-     <g>
-       <g id="man-bike">
-         <path id="Vector_2" opacity="0.1" d="M201.091 276.806L179.893 255.76L173.269 262.513L194.467 283.559L201.091 276.806Z" fill="black" />
```

Comment:
Why is this removed?

Reply:
because it was making the navbar long which made white space in this sides and also along the page and horizontal scroll ,also this element  wasn't used anywhere

Reply:
This element is used in the sidebar. Please have a look at the mobile screen sizes using Chrome Tools.

Reply:
ok i will add it